tests/integration_tests/performance/test_net_latency.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the network latency of a Firecracker guest."""
+
+import re
+import json
+import platform
+import pytest
+
+import host_tools.network as net_tools
+
+# Number of requests issued.
+REQUESTS = 500
+
+# Interval time between requests.
+INTERVAL = 0.2
+
+PING = 'ping -c {} -i {} {}'
+
+FLOAT_REG = r'[+-]?[0-9]+\.[0-9]+'
+
+HEAD = 'Min          Avg    Max      Mdev'
+
+GOLDFILE = 'goldfile.json'
+
+TEST_NAME = 'network_latency'
+
+
+def do_ping_on_guest(target, ssh_connection):
+    """Ping the @target on the guest."""
+    cmd = PING.format(REQUESTS, INTERVAL, target)
+    _, stdout, stderr = ssh_connection.execute_command(cmd)
+
+    assert stderr.read() == ''
+
+    return stdout.read()
+
+
+def get_statistics(output):
+    """Get statistics out of a ping result."""
+    output = output.split('\n')[-2]
+    return re.findall(FLOAT_REG, output)
+
+
+def check_results(result):
```

Comment:
would be also nice to test the p90 or something like that. average can be misleading. Also should check for lost packets. What happens if only one of the pings worked and the time for that is better than gold?

Reply:
That's a good point, I will do this check and add some perencetiles too